pepito_alonajane/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pepito, Alona Jane Navarro</title>
+    <link href="styles.css" rel="stylesheet"/>
+  </head>
+  <body>
+    <div class="box">
+      <h1 class="box-heading">Alona Jane Navarro Pepito </h1>
+        <p class="box-paragraph">Welcome to my website!</p>
+          <div class="container">
+           <div class="section">
+             <h2>My interests and hobbies</h2>
+                <ul>
+                  <li>I love to cook</li>
+                  <li>I love to watch movies</li>
+                  <li>I like to go to new places</li>
+                  <li>I love cats</li>
+                  <li>I love listening songs</li>
+                </ul>
+           </div> 
+          </div>
+      <div class="container">
+        <div class="section">
+         <h2>My works</h2>
+           <ol>
+            <a href="countries.html" target="_blank">
+                    Search Countries all over the World.</a>
+           </ol>
+        </div>
+      </div>
+    <div class="container">
+      <div class="section">
+         <h2>My Goals</h2>
+            <ul>
+              <li>Finish my education.</li>
+              <li>Get a good job.</li>
+              <li>Open a Restaurant/Bakery</li>
+              <li>Give back to my parents.</li>
+              <li>Financial Stability.</li>
+            </ul>
+      </div>
+    </div>
+     <div class="container">
+       <div class="section">
+         <h2>Comments</h2>
+            <button id="sort_asc_btn">
+            Sort by Date Ascending</button>
+            <button id="sort_desc_btn">
+               Sort by Date Descending</button>
+              <div id="commentsSection"></div>
```

Comment:
IDs should be written in snake case. All lower case separated by underscore.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5